src/transformations/remove-modifiers.js

```diff
@@ -35,6 +35,7 @@ function removeModifiers(rules) {
         modified = ruleUtils.removeModifier(props, 'document') || modified;
         modified = ruleUtils.removeModifier(props, 'doc') || modified;
         modified = ruleUtils.removeModifier(props, 'popup') || modified;
+        modified = ruleUtils.removeModifier(props, 'network') || modified;
```

Comment:
Validation rejects IP rules. Not sure if it will help.

Reply:
I took a look into how `||213.8.137.51^` and `||167.71.252.38^` successfully got accepted by validation. From what I could tell, it turned out their source entries were raw IPs, which seemingly had to have been accepted by the automated validation and only then had `||` and `^` appended to it by the Hostlist Compiler.

* https://github.com/AdguardTeam/AdguardFilters/blob/2c75320c9080a82bd7ce24f03fa47dbedbda9e20/BaseFilter/sections/adservers.txt#L30
* https://github.com/easylist/EasyListHebrew/blob/master/adguard_hosts.txt#L10

So the PR would as a makeshift temporary fix be able to make use of `$network` entries in AdGuard Base, which are practically always written as raw IPs to the best of my current knowledge. Sure this raises fair questions such as why the validator only checks the TLDs of adblocker-syntax entries and not raw IPs, but from my research that would be a separate aspect to fix.